framework/wazuh/cluster/cluster.py

```diff
@@ -456,8 +456,9 @@ def unmerge_agent_info(merge_type, path_file, filename):
             try:
                 st_size, name, st_mtime = header[:-1].split(' ', 2)
                 st_size = int(st_size)
-            except ValueError:
-                raise Exception("Malformed agent-info.merged file")
+            except ValueError as e:
+                logger.error("Malformed agent-info.merged file ({}). Parsed line: {}".format(e, header))
```

Comment:
```suggestion
                logger.error("Malformed agent-info.merged file ({str(e)}). Parsed line: {header}")
```

Reply:
Furthermore, I am wondering if this message should be tagged as 'error' or just as a 'warning'. Thoughts?

Reply:
Done!